File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/GroupBoxEnabled.vb
Namespace CompanionKit.Controls.GroupBox.Features

	Public Class GroupBoxEnabled

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

        End Sub
        ''' <summary>
        ''' Handles CheckedChanged event of the CheckBox control
        ''' </summary>
        Private Sub Enable_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles mobjChkEnable.CheckedChanged
            ''Switch GroupBox Enable property according to the CheckBox state
            Me.mobjGrpControls.Enabled = Me.mobjChkEnable.Checked
        End Sub

	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/GroupBoxEnabled.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.GroupBox.Features
{
	public partial class GroupBoxEnabled : UserControl
	{
		public GroupBoxEnabled()
		{
			InitializeComponent();
		}

        /// <summary>
        /// Handles CheckedChanged event of the CheckBox control
        /// </summary>
		private void Enable_CheckedChanged(object sender, EventArgs e)
		{
            //Switch GroupBox Enable property according to the CheckBox state
			mobjGrpControls.Enabled = mobjChkEnable.Checked;
		}
	}
}